src/sensor/controllers/sensor.controller.ts

```diff
@@ -0,0 +1,88 @@
+import { v4 as uuidv4 } from 'uuid';
+import { config } from '../../../config';
+import { SensorService } from '../services/sensor.service';
+import { RegisterSensorDto } from '../dtos/register-sensor.dto';
+import { SensorIdDto, LocationDto, DataStreamDto } from '../dtos/sensor.dto';
+import { ApiUseTags, ApiResponse, ApiOperation } from '@nestjs/swagger';
+import { Controller, Post, Param, Body, Delete, Put } from '@nestjs/common';
+import { UpdateSensorDetailsDto, TransferSensorOwnershipDto,
+  ShareSensorOwnershipDto } from '../dtos/update-sensor.dto';
+
+
+@Controller('sensor')
+@ApiUseTags('Sensor')
+export class SensorController {
+  constructor(private readonly sensorsService: SensorService) {}
+
+  @ApiOperation({ title: 'Register Sensor' })
+  @ApiResponse({ status: 200, description: 'Register Sensor.' })
+  @Post()
+  async createSensor(@Body() sensorDto: RegisterSensorDto): Promise<RegisterSensorDto> {
+    const id = uuidv4();
```

Comment:
Top! Goed idee om UUIDv4 te gebruiken. Daar hebben we het niet over gehad, maar had ik al wel in gedachten 😃